docs/sources/developers/plugins/_index.md

```diff
@@ -6,77 +6,51 @@ title: Build a plugin
 
-## Get started
+Many plugins are available in the [Catalog](https://grafana.com/grafana/plugins/) and others are built by developers for private use or to contribute them to the open source community. 
 
-The easiest way to start developing Grafana plugins is to use the Grafana [create-plugin tool](https://www.npmjs.com/package/@grafana/create-plugin).
+If you are a Grafana plugin developer or want to become one, this documentation is for you. Use it to get productive quickly, follow along with step-by-step tutorials, or you can refer back to it as you develop.  
 
-Open the terminal, and run the following command in your [plugin directory]({{< relref "../../setup-grafana/configure-grafana/#plugins" >}}):
+## Plugin basics
 
-```bash
-npx @grafana/create-plugin@latest
-```
+These are among the types of plugins you can create:
 
-Follow the questions and you will have a starter plugin ready to develop.
+- **Panel plugins** - Visualize data and navigating between dashboards.
+- **Data source plugins** - Link to new databases or other sources of data.
+- **App plugins** - Bundle panels and data sources together to create a better user experience.
+- **Other plugins** - Visualize data source logs or streaming data sources.
 
-If you want a more guided introduction to plugin development, check out our tutorials:
+The possibilities are as endless as your imagination, and you can use this guide to help make your dreams become reality. 
 
-- [Build a panel plugin](/tutorials/build-a-panel-plugin/)
-- [Build a data source plugin](/tutorials/build-a-data-source-plugin/)
+   Note: To learn more about the types of plugins you can build, refer to the [Plugin Overview]({{< relref "../../administration/plugin-management/" >}}) documentation.
 
-## Go further
-
-Learn more about specific areas of plugin development.
-
-### Tutorials
-
-If you're looking to build your first plugin, check out these introductory tutorials:
-
-- [Build a panel plugin](/tutorials/build-a-panel-plugin/)
-- [Build a data source plugin](/tutorials/build-a-data-source-plugin/)
-- [Build a data source backend plugin](/tutorials/build-a-data-source-backend-plugin/)
-
-Ready to learn more? Check out our other tutorials:
+## Contents of this developer's guide
 
-- [Build a panel plugin with D3.js](/tutorials/build-a-panel-plugin-with-d3/)
+The following topics give you an idea of what to expect from this guide:
 
-### Guides
+- [**Get started with plugins** - The easiest way to start developing Grafana plugins is to use the Create-plugin tool. Walk step by step for downloading the tool and using it to get your plugin up and running as fast as possible. 
+- [**Introduction to plugin development** - Ground yourself by learning the key concepts involved in Grafana plugin development. For example, learn about frontend and backend development processe, data frames, and error handling. 
```

Comment:
Shouldn't all these elements be links to their respective page?

Reply:
Done in PR 67070.